src/DynamoCoreWpf/Views/PackageManager/Controls/PackageManagerPackagesControl.xaml

```diff
@@ -276,7 +278,6 @@
                                                                         Visibility="{Binding CanInstall, Converter={StaticResource BoolToVisibilityCollapsedConverter}}"
                                                                         BorderThickness="0"
                                                                         Cursor="Hand"
-                                                                        Click="DropDownInstallButton_OnClick"
```

Comment:
so this fails to attach because the install button is not loaded?

Reply:
Just for context initially, we were setting directly the Command binding. That would trigger the same error until eventually, the binding would work when the button gets initialized.

Reply:
so what about instead of hooking up Loaded to `InstallButton` - what about attaching to the Loaded Event on the `dropDownInstallButton` - it seems it might be simpler at that point to attach the click handler - is that not the case?

Reply:
I tried that first, and that throws the same way as attaching the Click event. Only then I tried hooking to the parent button, and that did resolve the issue.